udata/core/dataset/models.py

```diff
@@ -151,13 +151,22 @@ def check_availability(self, group):
     def is_available(self):
         return self.check_availability(group=None)
 
+    @property
+    def permalink(self):
+        '''
+        Permanent link to the latest version of this resource.
+
+        If this resource is updated and `url` change, this property won't.
+        '''
+        return url_for('datasets.resource', id=self.id, _external=True)
+
     @cached_property
     def json_ld(self):
 
         result = {
             '@type': 'DataDownload',
             '@id': str(self.id),
-            'url': self.url,
+            'url': self.permalink,
```

Comment:
No we can't, it follows the schema specifications:
- https://schema.org/Dataset
- https://schema.org/DataDownload